src/backend/input.rs

```diff
@@ -216,6 +231,9 @@ pub trait InputBackend: Sized {
     /// Get current `InputConfig`
     fn input_config(&mut self) -> &mut Self::InputConfig;
 
+    /// Called to inform the Input backend about a new focused Output for a `Seat`
+    fn set_output_metadata(&mut self, seat: &Seat, output: &Output);
```

Comment:
Thinking about it, I would need to re-implement the transform methods in that case, as there is no generic representation.
I think we should use traits for these events.

Should I make everything event based in that case including tablet,keyboard and such?